Doc/whatsnew/3.12.rst

```diff
@@ -1289,8 +1289,9 @@ Deprecated
 
 * :mod:`typing`:
 
-  * :class:`typing.Hashable` and :class:`typing.Sized` aliases for :class:`collections.abc.Hashable`
-    and :class:`collections.abc.Sized`. (:gh:`94309`.)
+  * The :class:`typing.Hashable` and :class:`typing.Sized` aliases for
+    :class:`collections.abc.Hashable` and :class:`collections.abc.Sized` are deprecated and may be
+    removed in a future version of Python. (:gh:`94309`.)
```

Comment:
While it's true that we _may_ remove them in a future version of Python, that goes for pretty much anything in the standard library (whether deprecated or not!). If we ever decide we want to remove them, we'll announce it properly in "What's New" and emit deprecation warnings for at least two Python versions prior to removing them, as laid out in https://docs.python.org/3/library/typing.html#deprecated-aliases. As such, I think the "may be removed in a future version of Python" probably adds more uncertainty and fear than it adds clarity. I think I'd just omit it, personally.

Maybe something like this?

```suggestion
  * :class:`typing.Hashable` and :class:`typing.Sized`, aliases for
    :class:`collections.abc.Hashable` and :class:`collections.abc.Sized`
    respectively, are deprecated (:gh:`94309`).
```

Reply:
Updated

Reply:
Thanks! Nit for future reference: please avoid force-pushes where possible in CPython PRs, once somebody's started reviewing the patch :-)

It doesn't matter much for a small PR like this, but in general it makes it harder for reviewers to see what changes since they last reviewed -- the GitHub UI is really bad for force pushes. We squash-merge all PRs anyway, so there's no need to worry about a messy commit history :-)

Reply:
Sure. Thanks for letting me know!